historyuploader/src/main/java/com/twoez/historyuploader/Uploader.java

```diff
@@ -0,0 +1,59 @@
+package com.twoez.historyuploader;
+
+
+import com.twoez.common.SQLConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import picocli.CommandLine;
+
+import java.io.IOException;
+import java.sql.Timestamp;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+@CommandLine.Command(name = "Uploader", mixinStandardHelpOptions = true, version = "0.1")
+public class Uploader implements Callable<Integer> {
+
+    private Logger logger = LoggerFactory.getLogger(Uploader.class);
+
+    @CommandLine.Option(names = {"-d","--db_type"}, required = true, description = "Type of database where you want to store your prices.\n" + "Currently available: [mysql]")
+
+    String dbType;
+
+    @CommandLine.Option(names = {"-n", "--db_name"},required = true, description = "Name of database where to store prices")
+
```

Comment:
Redundant new lines - please remove. Looks awful when you open the class for the first time :)